src/sdk/pynni/nni/networkmorphism_tuner/README.md

```diff
@@ -0,0 +1,199 @@
+# Network Morphism Tuner on NNI
+
+## 1. Intorduction
+
+[Autokeras](https://arxiv.org/abs/1806.10282) is a popular automl tools using Network Morphism. The basic idea of Autokeras is to use Bayesian Regression to estimate the metric of the Neural Network Architecture. Each time, it generates several child networks from father networks. Then it uses a naïve Bayesian regression estimate its metric value from history trained results of network and metric value pair. Next, it chooses the the child which has best estimated performance and adds it to the training queue. Inspired by its work, we implement Network Morphism in our NNI platform.
```

Comment:
It would be better to mention that this tuner leverages autokeras's code.

Reply:
Thank you. I add the description about it later.